SentryTestUtils/Dynamic/Dynamic.swift

```diff
@@ -283,7 +283,9 @@ extension Dynamic {
         
         var storedSize = 0
         var storedAlignment = 0
+        //swiftlint:disable force_unwrapping
         NSGetSizeAndAlignment(invocation.returnType!, &storedSize, &storedAlignment)
+        //swiftlint:enable force_unwrapping
```

Comment:
This file must have been added before we started using swiftlint. because i renamed the file, it got linted. I don't want to change the logic in this file as part of this PR.